reflex/event.py

```diff
@@ -668,19 +667,23 @@ def call_script(
         callback_kwargs = {
             "callback": f"({arg_name}) => queueEvents([{format.format_event(event_spec)}], {constants.CompileVars.SOCKET})"
         }
+    wrapped_code = f"""
```

Comment:
I think this might be a way to catch frontend errors and send it to the backend, I might be wrong.